documentation/developer-guide/modules/tooling-guide/pages/bamm-cli.adoc

```diff
@@ -126,3 +176,46 @@ _models root_
 The name of the directory shown as _models root_ above can be chosen freely. The BAMM CLI will resolve the file path
 relative to the input file by following the folder structure described above. Each of the files in the `1.0.0` directory
 should therefore have an empty prefix declaration such as `@prefix : <urn:bamm:com.mycompany.myproduct:1.0.0#>`.
+
+
+[[Examples]]
+== Running Examples
+=== Validation
+1.	Identify the path to the model which is to be validated (Note: Here, Movement.ttl as an example)
+
+image::../images/8_ConfirmModelFile.png[]
+
+2.	Add further options to the CLI start including the path to the model. E.g Java -jar bamm-cli-1.02.jar -i  …io/openmanufacturing/1.0.0/Movement.ttl
```

Comment:
Add another full stop and a comma after "E.g", lowercase "java" (because it's a command here), surround with backticks and replace or remove the "…":

E.g., \`java -jar bamm-cli-{sds-sdk-version}.jar -i ./io/openmanufacturing/1.0.0/Movement.ttl`